### PR TITLE
Turned project into a python CLI package

### PR DESCRIPTION
**Pull request by peterHoburg**
I added a real setup.py file, github action, and changed the docs. I did NOT change any of the functional code in main.py, just moved it around. 

After this is merged you can simply go to https://pypi.org/ create an account, add your pypi account and username to this GitHub project by going to 

settings -> secrets -> "New repository secret". 

Then add `PYPI_PASSWORD` and `PYPI_USERNAME` as secrets. 

When you have done that you can simply make a new release and the Github action will take your Pypi secrets, build your python package, and upload it to Pypi! 

After that, anyone can get your package by `pip install mc-server-python`. 

Let me know if you have any questions. I am happy to walk you through everything I did.

### CODE REVIEW

**Comment by Yot360** [comment]
That's really cool! I appreciate the help/guiding.
I will make sure to do that.

**Comment by Yot360** [comment]
So I followed what you told me but the project doesn't seems to add in PyPi, so not available with pip. I added secrets with PYPI_PASSWORD and PYPI_USERNAME as name and my password/user as value.

**Comment by peterHoburg** [comment]
> So I followed what you told me but the project doesn't seems to add in PyPi, so not available with pip. I added secrets with PYPI_PASSWORD and PYPI_USERNAME as name and my password/user as value.

@Yot360 
You need to verify your Pypi email address. You can find the log for your issue under "Actions" 

https://github.com/Yot360/mc-server-python/runs/2643075058?check_suite_focus=true

**Comment by Yot360** [comment]
> > So I followed what you told me but the project doesn't seems to add in PyPi, so not available with pip. I added secrets with PYPI_PASSWORD and PYPI_USERNAME as name and my password/user as value.
> 
> 
> 
> @Yot360 
> 
> You need to verify your Pypi email address. You can find the log for your issue under "Actions" 
> 
> 
> 
> https://github.com/Yot360/mc-server-python/runs/2643075058?check_suite_focus=true

Oh, thank you, let me try again.

**Comment by Yot360** [comment]
My email is set as primary/verified but the same error shows up in logs.
Screenshot from 2021-05-22 01-30-07

**Comment by peterHoburg** [comment]
> My email is set as primary/verified but the same error shows up in logs.
> Screenshot from 2021-05-22 01-30-07

@Yot360 
The name was already taken. I made a new PR. It should fix it.
